src/test/components/__snapshots__/MarkerChart.test.js.snap

```diff
@@ -1361,7 +1361,7 @@ Array [
     "rect",
     88,
     17.5,
-    130.25,
+    130.5,
```

Comment:
In this test, the DPI is 2, that's why rounding can end to a value with `0.5` digit.